src/data-availability/BlobPointer.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity ^0.8.0;
+
+import "./MerkleProofInput.sol";
+struct BlobPointer {
```

Comment:
this structure is very tailored for avail, is the goal here to have a seperate nitro contract version with avail ? if so no worries, but if the goal is to enable avail as an optional opt in service this design will be very restrictive for expansion